README.md

```diff
@@ -141,7 +141,7 @@ helm install tetragon cilium/tetragon -n kube-system
 kubectl rollout status -n kube-system ds/tetragon -w
 ```
 
-By default, kube-system pods are filtered for the examples below we use the demo
+By default, kube-system pods are filtered for the examples below. We use the demo
```

Comment:
I believe the `for the examples below` clause belongs in the second sentence (and it should be followed by a comma as it's an independent clause).

Reply:
changed in 876334e8 + rebased